Gemfile

```diff
@@ -63,3 +63,4 @@ gem 'scalarm-database', '>= 0.3.1', git: 'git://github.com/Scalarm/scalarm-datab
 gem 'scalarm-service_core', '~> 0.9.1', git: 'git://github.com/Scalarm/scalarm-service_core.git'
 
 
+gem 'rinruby'
```

Comment:
Jeśli RinRuby nie jest używane, to należy usunąć zależność od niego. Tak samo automatyczna instalacja R libraries - jeśli są one instalowane dla RinRuby to potem działają z Rscript?

Instalację r libraries należy przenieść raczej do pierwszego uruchomienia supervisora - bazowa usługa powinna mieć jak najmniej zależności, a supervisory powinny być w formie pluginu - dodanie lub usunięcie supervisora nie powinno modyfikować bazowej usługi. Na razie wyjątkiem są testy.